index.html

```diff
@@ -5,16 +5,185 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
-    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="shortcut icon" type="image/icon"  href="image/icon.PNG">
+ 
 
-    <title>Responsive grid project</title>
+    <title>Responsive grid project Meisam</title>
 </head>
-
 <body>
+<!-- Start Header -->
+<header id="header">
+    <div class="logo" data-text="img"><img src="Image/Logo.jpg" 
+     alt="logo" title="Glasses"></div></div>
 
-    <!-- Add your markup here -->
+    <nav id="nav">
+    <label for="hamburger"></label>
+    <ul>
+        <li><a href="">Home</a></li>
+        <li><a href="">Women</a></li>
+        <li><a href="">Men</a></li>
+        <li><a href="">Brand</a></li>
+        <li><a href="">About</a></li>
+    </ul>
+    </nav>
+ </header>
+ <main>
+<section>
+   <div class="image-main  col-12">
+    <div id="hero">
+        <h1>
+            DESIGNER SALE 
+            <p>FURTHER REDUCTIONS</p>
+            50% OFF !
+        </h1>
+    </div>
+    <button class="button" type="button">
+        Shop Now
+    </button>
+   </div>
+</section>
 
-</body>
+<div class="container col-12  ">
+    <div class="row">   
+      
+        <div class="sm-col-11 md-col-6 lg-col-4 xl-col-3">
+         <div class="gallery">
+          <a target="_blank" href="1.jpg">
+          <img src="Image/1.jpg" alt="Forest" >
+          </div>
+          <div class="desc">Add a description of the image here</div>
+        </div>
+ 
+        <div class="sm-col-11 md-col-6 lg-col-4 xl-col-3">
+          <div class="gallery">
+           <a target="_blank" href="2.jpg">
+           <img src="Image/2.jpg" alt="Northern Lights" >
+          </div>
+          <div class="desc">Add a description of the image here</div>
+        </div>
+  
+        <div class="sm-col-11 md-col-6 lg-col-4 xl-col-3">
+         <div class="gallery">
+          <a target="_blank" href="3.jpg">
+          <img src="Image/3.jpg" alt="Northern Lights" >
+         </div>
+         <div class="desc">Add a description of the image here</div>
+        </div>
+
+        <div class="sm-col-11 md-col-6 lg-col-4 xl-col-3">
```

Comment:
It might look better if you keep the image description below the image for `sm` size as well.